### PR TITLE
Footer Changes

### LINKED ISSUE #32: [DOCS] Footer
### Description

Improve footer section by adding logo and important links mentioned in WasmEdge website.  I am working on this.

### PR DESCRIPTION
**Pull request by Sukriti-sood**
Added logo and links that are also mentioned in WasmEdge [website](https://wasmedge.org/). 

Close #32

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -118,26 +118,45 @@ const config = {
...
+                style: 'light',
                 links: [
                     {
-                        title: 'Docs',
+                        title: 'Resources',
                         items: [
                             {
-                                label: 'Develop',
-                                to: '/develop/overview',
+                                label: 'Github',
```
Comment (kokamkarsahil) [inline docusaurus.config.js:138]: Link to GitHub is already present at global header.

File: docusaurus.config.js [hunk 2]
```diff
@@ -118,26 +118,45 @@ const config = {
...
+                                to: 'https://github.com/WasmEdge/WasmEdge',
                             }, {
-                                label: 'Embed',
```
Comment (kokamkarsahil) [inline docusaurus.config.js:134]: Was there any reason for removing guides links?

File: docusaurus.config.js [hunk 3]
```diff
@@ -146,10 +165,18 @@ const config = {
...
+                                to: 'https://slack.cncf.io/'
+                            },
+                            {
+                                label: 'Community Meeting',
+                                to: 'https://docs.google.com/document/d/1iFlVl7R97Lze4RDykzElJGDjjWYDlkI8Rhf8g4dQ5Rk/edit?usp=sharing'
+                            }
                         ],
                     },
                 ],
-                copyright: `Copyright © ${new Date().getFullYear()} WasmEdge. Built with Docusaurus.`,
```
Comment (kokamkarsahil) [inline docusaurus.config.js:156]: Keeping _"Built with Docusaurus"_  will be in good faith for Open Source community.

File: docusaurus.config.js (hunk 1 above)
Comment (Sukriti-sood) [reply docusaurus.config.js:138]: Users should be able to access all vital links from the footer.

File: docusaurus.config.js (hunk 2 above)
Comment (Sukriti-sood) [reply docusaurus.config.js:134]: Yes, this is a mistake.

File: docusaurus.config.js (hunk 3 above)
Comment (Sukriti-sood) [reply docusaurus.config.js:156]: Copyright should be small and concise.

Comment (adithyaakrishna) [reply docusaurus.config.js:156]: @Sukriti-sood Removing docusaurus is like basically going against the OSS principles. We are using a OSS framework for the site and the creators/maintainers of docusaurus project should get their credit as well. IMHO we should keep the `Built with docusaurus` message

Comment (Sukriti-sood) [reply docusaurus.config.js:156]: Sorry @adithyaakrishna @kokamkarsahil , I will revert this change. I understand the principles of OSS and not in against of giving credits to creators.

**Comment by adithyaakrishna** [comment]
LGTM 🎉

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
LGTM

**Pull request merged**